src/OpenTelemetry.Instrumentation.EventCounters/EventCountersInstrumentationOptions.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="EventCountersInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Instrumentation.EventCounters;
+
+/// <summary>
+/// EventCounterListener Options.
+/// </summary>
+public class EventCountersInstrumentationOptions
+{
+    private readonly HashSet<string> eventSourceNames = new();
+
+    /// <summary>
+    /// Gets or sets the subscription interval in seconds.
+    /// </summary>
+    public int RefreshIntervalSecs { get; set; } = 1;
```

Comment:
might need a better description to convey the fact that this is really the interval at which we refresh/read values from eventcounter.